docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+# For running locally
+
+version: '3'
+services:
+  telegram-bot:
+    image: docker.pkg.github.com/notify-watcher/telegram-bot/telegram-bot
+    environment:
+      - NODE_ENV=development
```

Comment:
Why?

Reply:
because this is for running locally for development (if you want to develop using docker)